doc/languages-frameworks/rust.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Rust
+author: Matthias Beyer
+date: 2017-03-05
+---
+
+# User's Guide to the Rust Infrastructure
+
+Rust is packaged with nixpkgs, although there are no rust crates packaged in
+nixpkgs.
+Therefor, one has to use `cargo` directly.
```

Comment:
I have not tried it yet, but maybe it might be worth to add a reference to https://github.com/fractalide/nixcrates